frontend/src/components/grantaccess/form/ForestClientInput.vue

```diff
@@ -27,14 +27,22 @@ const emit = defineEmits([
 const forestClientNumbersInput = ref('');
 const forestClientData = ref<FamForestClient[]>([]);
 const forestClientNumberVerifyErrors = ref([] as Array<string>);
+const verifiedNumbers: Set<string> = new Set();
 
 const verifyForestClientNumber = async (forestClientNumbers: string) => {
     forestClientNumberVerifyErrors.value = [];
 
     // split by commas and spaces
-    let forestNumbers = forestClientNumbers.split(',').map(num => num.trim());
+    let forestNumbers = forestClientNumbers.split(',').map((num) => num.trim());
 
     for (const item of forestNumbers) {
```

Comment:
and let's rename this "item", I think this is "forestClientNumber", type string?

Reply:
sure thing!